src/app/beer_garden/app.py

```diff
@@ -133,7 +151,20 @@ def handle_event(event):
         if event.garden == beer_garden.config.get("garden.name"):
             # Start local plugins after the entry point comes up
             if event.name == Events.ENTRY_STARTED.name:
-                PluginManager.instance().scan_path()
+                beer_garden.local_plugins.manager.lpm_proxy.scan_path()
+
+            elif event.name == Events.INSTANCE_INITIALIZED.name:
+                beer_garden.local_plugins.manager.lpm_proxy.handle_associate(event)
+            elif event.name == Events.INSTANCE_STARTED.name:
+                beer_garden.local_plugins.manager.lpm_proxy.handle_start(event)
+            elif event.name == Events.INSTANCE_STOPPED.name:
+                beer_garden.local_plugins.manager.lpm_proxy.handle_stop(event)
+            elif event.name == Events.SYSTEM_REMOVED.name:
+                beer_garden.local_plugins.manager.lpm_proxy.remove_system(event.payload)
+            elif event.name == Events.SYSTEM_RELOAD_REQUESTED.name:
+                beer_garden.local_plugins.manager.lpm_proxy.reload_system(event.payload)
+            elif event.name == Events.SYSTEM_RESCAN_REQUESTED.name:
+                beer_garden.local_plugins.manager.lpm_proxy.scan_path()
```

Comment:
I don't know why this got pulled up. Why can't this stay with the Plugin Manager as a registered listener for events?

Reply:
Because the Plugin Manager now lives in its own process, and that process doesn't have any plumbing to listen for events. But this is still a WIP, a lot of this logic is going to change now (the plugin manager shouldn't need to care about events for the most part anymore).